benchmarks/setup/default/zeebe-values.yaml

```diff
@@ -11,12 +11,17 @@ cpuThreadCount: 4
 ioThreadCount: 4
 gatewayMetrics: true
 
-zeebeCfg: |
-  # custom TOML configuration
+gateway:
+  replicas: 1
+  logLevel: debug
+
+# Uncomment to add custom YAML configuration
+# This will overwrite anything at the application.yml location
+# zeebeCfg: {}
 
 # JavaOpts:
 # DEFAULTS
-JavaOpts: |
+JavaOpts: >-
```

Comment:
wouldn't just `>` be enough here?  Thanks for the reference btw interesting read.

Reply:
Yeah you're right, plus we don't expect any nested values, so :man_shrugging: